app/assets/javascripts/discourse/app/lib/transformer/registry.js

```diff
@@ -8,4 +8,6 @@ export const VALUE_TRANSFORMERS = Object.freeze([
   "header-notifications-avatar-size",
   "home-logo-href",
   "home-logo-image-url",
+  "category-description-text",
```

Comment:
This is just a minor nitpick really, but it would be nice to keep the items in alphabetic order.

Reply:
sure ✅